.travis.yml

```diff
@@ -90,17 +90,6 @@ jobs:
         - COMPILER="ccache /usr/bin/g++-5"
         - EXTRA_CXXFLAGS="-D_GLIBCXX_DEBUG"
 
-    # OS X using g++
-    - stage: Test different OS/CXX/Flags
-      os: osx
-      sudo: false
-      compiler: gcc
-      cache: ccache
-      before_install:
-        - HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache parallel
-        - export PATH=$PATH:/usr/local/opt/ccache/libexec
-      env: COMPILER="ccache g++"
```

Comment:
Yes, that's a good idea, but I do note that the other build sets: `EXTRA_CXXFLAGS="-Qunused-arguments -fcolor-diagnostics"` and `CCACHE_CPP2=yes` - both of which should IMHO be removed.

Reply:
My understanding was that for clang, you do really want CCACHE_CPP2=yes because of the way clang/llvm works, otherwise you get much poorer use of ccache. I haven't actually tested that myself though, so the world may well have moved on, but this http://peter.eisentraut.org/blog/2014/12/01/ccache-and-clang-part-3/ has some info and links to older info too.

Reply:
This may (still) be true for Linux, but for the `brew`-provided `ccache` it seems to be set to yes/true by default anyway. Output on my Mac, no tweaks over the default config:
```
$ ccache -p | grep run_second_cpp
(default) run_second_cpp = true
```